benchmarks/perfetto/install.py

```diff
@@ -0,0 +1,23 @@
+import os
+import sys
+import subprocess
+
+from . import s3_utils, BENCHMARK_ROOT
+
+TRACES = [
+    "torchbench_traces.tar.gz",
+]
+
+def download_traces_from_s3():
+    """Download trace to benchmarks/trace_analysis/.data"""
+    for trace in TRACES:
+        s3_utils.checkout_s3_data(trace, decompress=True)
+
+def install_deps(requirements_txt="requirements.txt"):
+    subprocess.check_call(
+        [sys.executable, "-m", "pip", "install", "-r", os.path.join(BENCHMARK_ROOT, requirements_txt)]
+    )
+
+if __name__ == "__main__":
+    download_traces_from_s3()
+    install_deps()
```

Comment:
Do you want to call `install_deps` before `download_traces_from_s3` and add `requests` to the requirements?

Reply:
fixed